src/main/java/sirius/web/controller/Message.java

```diff
@@ -76,14 +76,14 @@ public Message withHTMLMessage(String htmlMessage) {
         public Message withTextAndLink(String textMessage, String label, String link, @Nullable String icon) {
             if (Strings.isFilled(icon)) {
                 return new Message(type,
-                                   Strings.apply("<span>%s</span><a href=\"%s\"><i class=\"fa %s\"></i> %s</a>",
+                                   Strings.apply("<span class\"pr-2\">%s</span><a href=\"%s\"><i class=\"fa %s\"></i> %s</a>",
```

Comment:
Optional: Escaping of the quotes could potentially be omitted when using `'` instead.